src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Statistics/FootnoteController.cs

```diff
@@ -140,6 +141,15 @@ public async Task<ActionResult<FootnotesMetaViewModel>> GetFootnotesMeta(Guid re
                 .HandleFailuresOrOk();
         }
 
+        [HttpPatch("releases/{releaseId:guid}/footnotes")]
+        public async Task<ActionResult<Unit>> UpdateFootnotes(
+            Guid releaseId,
+            FootnotesUpdateRequest request)
+        {
+            return await _footnoteService.UpdateFootnotes(releaseId, request)
```

Comment:
I called it `UpdateFootnotes` to be consistent with `UpdateFilters` and `UpdateIndicators` which exist elsewhere in `TableBuilderMetaController`, and to be consistent with the url suffix `/footnotes`.

These methods could allow bulk updating more than just the order depending on what's contained in their body request types so that's why they are abstract.